src/ascent/runtimes/expressions/ascent_blueprint_architect.cpp

```diff
@@ -1417,32 +1396,29 @@ binning(const conduit::Node &dataset,
   conduit::Node res;
   res["value"].set(conduit::DataType::c_double(num_bins));
   double *res_bins = res["value"].value();
-  if(reduction_op == "cnt")
-  {
-#ifdef ASCENT_USE_OPENMP
-#pragma omp parallel for
-#endif
-    for(int i = 0; i < num_bins; ++i)
-    {
-      res_bins[i] = bins[i];
-    }
-  }
-  else if(reduction_op == "pdf")
+  if(reduction_op == "pdf")
   {
-    int n = 0;
+    double total = 0;
 #ifdef ASCENT_USE_OPENMP
 #pragma omp parallel for reduction(+ : n)
```

Comment:
There is no `n` since the previous line `int n = 0` seems to be gone.

Reply:
probably should be `reduction(+ : total)`

Reply:
thanks!